### ISSUE TITLE
[C2-M1 Page Structure] Screen reader use not suggested?

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-curricula#219]
> Ideas to Assess Knowledge for Module
> Short answer questions — Students are asked to explore a given web page with their screen reader of choice. Then they are asked to provide information about how many headings, regions, or links the page has. Assess students’ knowledge of how to use assistive technologies to orient and navigate web content.

I am not sure about suggesting that students use screen reader. I think it's important that all involved in web development understand the concept of how screen readers work. I do not think it's important that most people can use one themselves. It's actually fairly hard to learn to use a screen reader effectively, and I think encouraging it could backfire in some circumstances.

Also, how this assessment is written doesn't work. Students could answer questions about how many headings, regions, or links the page has without using a screen reader. So that would not "students’ knowledge of how to use assistive technologies to orient and navigate web content.".

**Comment by daniel-montalvo** [comment]
How about something along the lines of:

* Short answer questions &mdash; Students are asked about the mechanisms available for users of assistive technologies to move to next and previous heading and to show all the headings of a web page in an isolated list. Assess students’ knowledge of mechanisms of assistive technologies to move through headings.

With this change we remove that potential risk you mention, but I think we still capture the need for a developer to understand the concept of how screen readers in particular (and assistive technologies in general) work.

**Comment by shawna-slh** [comment]
This is the part that I think is relevant in a Developer course:
> identify how many headings, regions, or links the page has.

This:
> mechanisms available for users of assistive technologies to move to next and previous heading and to show all the headings of a web page in an isolated list. Assess students’ knowledge of mechanisms of assistive technologies to move through headings.

I mildy still think is something that they should have learned in the first Curricula, and should be a prerequisite for this course. However, I'm fine accepting the decision of the Task Force.

**Comment by nitedog** [comment]
I agree that the focus should not be making students expert assistive technology users. Yet I think that some *guided* exposure can be very beneficial and improve the understanding. I'd prefer a combination of both suggestions.

**Comment by daniel-montalvo** [comment]
Yesterday this was discussed in the [TF meeting](https://www.w3.org/2020/08/25-wai-curricula-minutes.html#item03).

There was agreement that some degree of exposure to assistive technology use is needed to understand the relevance of page structure, and even to quickly and efficiently test if some of the structural components are used adequately.

We also acknowledged that it could backfire if students feel they lose control of their devices when activating an assistive technology, such as a screen reader.

I was hesitant to include the guidance in the assessment, as it could render it very difficult to use in a non-guided scenario. But we also agreed that guidance may not necessarily be provided face-to-face, but some multimedia materials can be used.

We plan to move forward with the changes at
#244 
@slhenry Would you be comfortable with some degree of **guided** exposure?
Thanks.

**Comment by shawna-slh** [comment]
Yes! I +1ed the changes.
